invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/administration/components/SetQuotaForm.js

```diff
@@ -0,0 +1,162 @@
+/*
+ * // This file is part of Invenio-App-Rdm
+ * // Copyright (C) 2023 CERN.
+ * //
+ * // Invenio-App-Rdm is free software; you can redistribute it and/or modify it
+ * // under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { i18next } from "@translations/invenio_app_rdm/i18next";
+import { Formik } from "formik";
+import {
+  withCancel,
+  ErrorMessage,
+  TextField,
+  TextAreaField,
+  http,
+} from "react-invenio-forms";
+import { Form, Button, Modal } from "semantic-ui-react";
+import { NotificationContext } from "@js/invenio_administration";
+import * as Yup from "yup";
+
+export class SetQuotaForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      loading: false,
+      error: undefined,
+    };
+    this.tombstoneSchema = Yup.object({
+      quota_size: Yup.number().required().min(1),
+      max_file_size: Yup.number().required().min(1),
+      notes: Yup.string(),
+    });
+  }
+
+  componentWillUnmount() {
+    this.cancellableAction && this.cancellableAction.cancel();
+  }
+
+  static contextType = NotificationContext;
+
+  handleSubmit = async (values) => {
+    this.setState({ loading: true });
+    const { addNotification } = this.context;
+    const { resource, actionSuccessCallback, apiUrl } = this.props;
+    const payload = { ...values };
+    this.cancellableAction = withCancel(http.post(apiUrl, payload));
+    try {
+      await this.cancellableAction.promise;
+      this.setState({ loading: false, error: undefined });
+      addNotification({
+        title: "Success",
```

Comment:
question: why are notifications not translated? I see they are not translated in other parts of the administration panel either.